### DISCUSSION TITLE
New Server SSL certificate drom OU record

### DISCUSSION THREAD

**Discussion by puresight2021** [strongswan/strongswan#733] [Q&A]
We have on our server a configuration that uses the server certificate and the identity of the server:
leftcert=serverCert.pem

When we renew the SSL certificate, we got a new identity that does not include OU value:
The old one was: 
Subject: **OU = Domain Control Validate**d, CN = *.XXX.com
and the new one is :
Subject: CN = *.XXX.com

On the client, we set the set remote Id to be "OU = Domain Control Validated, CN = *.XXX.com" so now when we update the new certification, the isn't any match with the client request and the server configuration.
From what I understand from Godday that provided me with the SSL certification, the OU is not supporting anymore, but I have many installed clients that are sending the old remote-id with the OU record.

Is there a way to bypass this issue? Supporting the old clients with a new certificate that does not contain the OU record ?

Thank,
Royi

**Comment by tobiasbrunner** [comment]
Not if you can't change the client's config (and if you could, you could just change the identity). Because even if the server would accept the identity requested by the client, the client would later reject the server certificate as its configured identity doesn't match it.
